client/helpers.go

```diff
@@ -62,3 +63,36 @@ func nearExpiry(r *data.SignedRoot) bool {
 	plus6mo := time.Now().AddDate(0, 6, 0)
 	return r.Signed.Expires.Before(plus6mo)
 }
+
+func initRoles(kdb *keys.KeyDB, rootKey, targetsKey, snapshotKey, timestampKey data.PublicKey) error {
+	rootRole, err := data.NewRole("root", 1, []string{rootKey.ID()}, nil, nil)
```

Comment:
I'm always confused if the minVersion is 0 or 1. Also we still need to figure that out in VerifyRoot.